nullaway/src/main/java/com/uber/nullaway/fixserialization/FixSerializationConfig.java

```diff
@@ -110,6 +129,15 @@ public FixSerializationConfig(String configFilePath) {
       throw new IllegalStateException(
           "Error in the fix serialization configuration, suggest flag must be enabled to activate enclosing method and class serialization.");
     }
+    methodParamProtectionTestEnabled =
+        XMLUtil.getValueFromAttribute(document, "/serialization/paramTest", "active", Boolean.class)
+            .orElse(false);
+    paramTestIndex =
+        (int)
+            Math.round(
```

Comment:
Does XML somehow not support integers, so we have to do this rounding?

Reply:
Yes it does, actually I am only working with strings and it was my mistake from converting `json` implementation to `xml` and this was left from there. Thank you for noting it. Resolved it now. 1c9e0bfb3dd002cade3e0b372bf968eb615f0232